src/DevSoc.Eventathon.Attendance/AttendanceService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DevSoc.Eventathon.Attendance.Models;
+using DevSoc.Eventathon.Calendars.Models;
+
+namespace DevSoc.Eventathon.Attendance;
+
+public class AttendanceService : IAttendanceService
```

Comment:
I'm on the fence about whether this service is really needed - you could just do the mapping in the controller and have a direct dependency on the `IAttendanceRepository`, but I think this is okay.
Up to you whether you keep the service or not.